src/includes/enriching-events/attachment-init-with-path/dart.mdx

```diff
@@ -0,0 +1,6 @@
+```dart
+// Please note that this is not available on web platforms
```

Comment:
How should notes like this be added?

Reply:
we can add the `<Note>`  or `<Warning>` before the code snippet maybe, we have examples across the docs